gradle/versions.gradle

```diff
@@ -10,19 +10,19 @@ ext {
     gsonVersion = '2.10.1'
     httpClientVersion = '4.5.14'
     httpCoreVersion = '4.4.16'
-    commonsCodecVersion = '1.16.0'
+    commonsCodecVersion = '1.16.1'
     slf4jVersion = "1.7.36"
     snakeYaml = "2.2"
     jacksonCoreVersion = '2.16.1'
     jacksonDatabindVersion = '2.16.1'
-    jsonPathVersion = "2.8.0"
+    jsonPathVersion = "2.9.0"
     junitVersion = "4.13.2"
     restAssuredVersion = "4.5.1"
     hamcrestJunitVersion = "2.0.0.0"
-    tomcatVersion = "9.0.84"
+    tomcatVersion = "9.0.85"
     activationApiVersion = "1.2.2"
     xmlBindApiVersion = "2.3.3"
-    objenesisVersion = "3.3"
+    objenesisVersion = "3.2"
```

Comment:
why the downgrade here?

Reply:
Good point! I don't know - a mistake. Thank you, reverted back.